sqlmesh/core/state_sync/engine_adapter.py

```diff
@@ -889,24 +878,24 @@ def _snapshot_id_filter(
                 exp.Tuple,
                 exp.convert(
                     (
-                        exp.to_column(f"{fq_table_name}.name"),
```

Comment:
So as far as I remember columns must be qualified in the filter because of of a join using `ON`. At least for MSSQL. @treysp please correct me if I'm wrong.

Reply:
they are qualified using aliases